mcux/CMakeLists.txt

```diff
@@ -61,6 +61,10 @@ add_subdirectory_ifdef(
   middleware/wireless/framework_5.3.3
   )
 
+if (${MCUX_DEVICE} MATCHES "MIMX8MN6")
+	zephyr_library_sources(devices/${MCUX_DEVICE}/system_MIMX8MN6_cm7.c)
```

Comment:
Can you move this down and make it similar to how the CONFIG_SOC_MIMXRT685S_CM33 is done.